ui/app/first-time/init-menu.js

```diff
@@ -21,6 +21,7 @@ function mapStateToProps (state) {
     // state from plugin
     currentView: state.appState.currentView,
     warning: state.appState.warning,
+    forgottenPassword: state.appState.forgottenPassword,
```

Comment:
I swear that we don't need to use `state.appState.forgottenPassword`.  I wouldn't block over this, but I would prefer if this line was:
```
  forgottenPassword: state.metamask.isInitialized,
```